### PR TITLE
Add 6.3 to wonfix_versions in tikv repo.

### PR DESCRIPTION
**Pull request by MimeLyc**

### CODE REVIEW

**Comment by MimeLyc** [comment]
cc @BusyJay

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -1875,3 +1874,30 @@ ti-community-issue-triage:
...
+      - "5.1"
+      - "5.2"
+      - "5.3"
+      - "5.4"
+      - "6.0"
+      - "6.1"
+      - "6.2"
+      - "6.3"
+    wontfix_versions:
+      - "3.0"
```
Comment (wuhuizuo) [inline prow/config/external_plugins_config.yaml:1891]: the item is not in `maintain_versions`, does it need to be contained in the `wontfix_versions` list? @MimeLyc

**Comment by wuhuizuo** [comment]
rest LGTM

File: prow/config/external_plugins_config.yaml [hunk 2]
```diff
@@ -1875,3 +1874,29 @@ ti-community-issue-triage:
...
+      - "5.0"
+      - "5.1"
+      - "5.2"
+      - "6.0"
+      - "6.2"
+      - "6.3"
+    affects_label_prefix: "affects-"
+    may_affects_label_prefix: "may-affects-"
+    linked_issue_needs_triage_label: "do-not-merge/needs-triage-completed"
+    need_cherry_pick_label_prefix: "needs-cherry-pick-"
```
Comment (wuhuizuo) [inline prow/config/external_plugins_config.yaml:1900]:
```suggestion
    need_cherry_pick_label_prefix: "needs-cherry-pick-release-"
```

tikv/tikv is using cherry-pick function in TiChiBot

File: prow/config/external_plugins_config.yaml [hunk 3]
```diff
@@ -1823,7 +1823,6 @@ ti-community-issue-triage:
     need_cherry_pick_label_prefix: "needs-cherry-pick-release-"
     status_target_url: "https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/cherrypick-a-pr.html#pass-triage-complete-check"
   - repos:
-      - pingcap/br
```
Comment (wuhuizuo) [inline prow/config/external_plugins_config.yaml:1826]: ？should delete `tikv/tikv`?

Comment (wuhuizuo) [reply prow/config/external_plugins_config.yaml:1826]: @MimeLyc

**Review by wuhuizuo — CHANGES_REQUESTED** [review]

File: prow/config/external_plugins_config.yaml (hunk 3 above)
Comment (MimeLyc) [reply prow/config/external_plugins_config.yaml:1826]: my fault. already resolved.

**Review by wuhuizuo — APPROVED** [review]

**Pull request merged**